packages/frontend/src/components/message/Message.tsx

```diff
@@ -284,6 +294,12 @@ function buildContextMenu(
       label: tx('reply_noun'),
       action: setQuoteInDraft.bind(null, message.id),
     },
+    showEdit && {
+      // Not `tx('edit_message')`.
+      // See https://github.com/deltachat/deltachat-desktop/issues/4695#issuecomment-2688716592
+      label: tx('global_menu_edit_desktop'),
+      action: enterEditMessageMode.bind(null, message),
+    },
```

Comment:
let's move "Edit" after the "message interaction with others" block, so after reply/reply-privately/forward/react 

so, after "// Send emoji reaction"

order would be:

Reply
[Reply Privately]
Forward
React
[Edit]
Save
[Save As]
Copy Text/Link/Image
[Open Attachment]
[Add To Sticker Collection[
[Resend]
Info
Delete

(at some point, a separator or a submenu would be handy, but that is not in this PR)